pandas/core/groupby/base.py

```diff
@@ -12,11 +12,11 @@
 
 
 class GroupByMixin(object):
-    """ provide the groupby facilities to the mixed object """
+    """ Provide the groupby facilities to the mixed object. """
```

Comment:
Do you mind changing all the cases like this (quotes in the same line) to this:
`
"""
Provide the groupby facilities to the mixed object.
"""
`
This will fail the validation soon too, so if you don't mind changing the edited ones, it'd be great.

Reply:
Sorry, the formatting is slightly unclear - it shows no change from what I've already added in. Did you mean to say I should move it to a new line?

Reply:
sorry, I did a mistake with the fithib formatting, amd wasn't rendering as I intended. I editted the comment, you can see now what I meant